validator/client/metrics.go

```diff
@@ -190,6 +190,30 @@ var (
 			"pubkey",
 		},
 	)
+
+	// ValidatorSyncCommitteeGaugeVec used to keep track of whether validator is part of sync committee currently
+	ValidatorSyncCommitteeGaugeVec = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "validator",
+			Name:      "sync_committee",
+			Help:      "whether validator is part of sync committee currently",
+		},
+		[]string{
+			"pubkey",
+		},
+	)
+	// ValidatorNextSyncCommitteeGaugeVec used to keep track of whether validator is part of next sync committee
+	ValidatorNextSyncCommitteeGaugeVec = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "validator",
+			Name:      "next_sync_committee",
```

Comment:
Naming wise, I would suggest something like 

- validator_sync_committee_current
- validator_sync_committee_next

The idea behind the naming is that it has a common prefix and makes it easier to find these metrics in some cases